Gruntfile.js

```diff
@@ -239,7 +240,7 @@ module.exports = function(grunt) {
 
     githooks: {
       all: {
-        'pre-commit': 'jshint cover'
+        'pre-commit': 'jshint'
```

Comment:
I think it does need to be removed since it's broken code, but in a separate commit/pr just for separation of concerns. I wasn't paying close attention when I included it in this commit. Are we wanting to recreate the test githook elsewhere, or simply remove it?